sklearn/feature_selection/from_model.py

```diff
@@ -43,21 +44,25 @@ def transform(self, X, threshold=None):
         X_r : array of shape [n_samples, n_selected_features]
             The input samples with only the selected features.
         """
+        msg = ("This %s instance is not properly "
+               "fitted yet." % (type(self).__name__))
+        check_is_fitted(self, ('coef_', 'feature_importances_'), msg=msg,
```

Comment:
Actually no... I just tried to retain the old messages wherever possible... I'll remove this?